website/content/docs/agent/autoauth/methods/approle.mdx

```diff
@@ -35,3 +35,62 @@ cached.
   containing the output of the secret ID retrieval endpoint for the role (e.g.
   `auth/approle/role/webservers/secret-id`) and the creation path for the
   response-wrapping token must match the value set here.
+
+## Example Configuration
+
+An example configuration, using approle to enable [auto-auth](/docs/agent/autoauth) and creating both a plaintext token sink and [encrypted token sink file](/docs/agent/autoauth#wrap_ttl), follows:
+
+```python
+pid_file = "./pidfile"
+
+vault {
+        address = "https://127.0.0.1:8200"
+}
+
+auto_auth {
+  method {
+    type      = "approle"
+
+    config = {
+      role_id_file_path = "roleid"
+      secret_id_file_path = "secretid"
+      remove_secret_id_file_after_reading = false
+    }
+  }
+
+    sink {
```

Comment:
Can you indent this consistently with `method`?

Reply:
Done